src/main/java/seedu/address/model/date/TrainingDate.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.date;
+
+import static java.util.Objects.requireNonNull;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Represents date used in attendance and performance recording.
+ */
+public class TrainingDate {
```

Comment:
Could this be used for storing their birthdays as well?

Reply:
if we decide to add a birthday attribute, we can certainly do so, however i think we are focusing on our main features now before we move on to the "nice to haves"